modules/sdk/src/main/scala/org/tessellation/sdk/domain/Daemon.scala

```diff
@@ -1,10 +1,16 @@
 package org.tessellation.sdk.domain
 
+import cats.FlatMap
 import cats.effect.kernel.Spawn
+import cats.syntax.flatMap._
 import cats.syntax.functor._
 
-trait Daemon[F[_]] {
+trait Daemon[F[_]] { self =>
   def start: F[Unit]
+
+  def combine(other: Daemon[F])(implicit F: FlatMap[F]): Daemon[F] = new Daemon[F] {
```

Comment:
How about making `Semigroup[Daemon]`?

Reply:
I was thinking about it as well, but I wasn't sure if that will follow the laws of the semigroup. Idk, to be honest. Also after @marcinwadon commented https://github.com/Constellation-Labs/tessellation/pull/216#discussion_r833123422 that we won't need an additional daemon, this `combine` method wouldn't be used anymore, so I was thinking about removing it altogether.